File: PRAK305-2510817220020-LailyAzizahIslami.c
#include <stdio.h>

int main() {
    int detik, hari, jam, menit, sisa;
    scanf("%d", &detik);

hari = detik / 86400;
sisa = detik % 86400;
jam = sisa / 3600;
sisa %= 3600;
menit = sisa / 60;
sisa %= 60;
if (hari > 0)
    printf("%d hari %02d:%02d:%02d", hari, jam, menit, sisa);
else
    printf("%02d:%02d:%02d", jam, menit, sisa);
    
return 0;
}

File: PRAK305-2510817220020-LailyAzizahIslami.py
detik = int(input())
hari = detik // 86400
sisa = detik % 86400
jam = sisa // 3600
sisa %= 3600
menit = sisa // 60
detik = sisa % 60

if hari > 0:
    print(f"{hari} hari {jam:02d}:{menit:02d}:{detik:02d}")
else:
    print(f"{jam:02d}:{menit:02d}:{detik:02d}")